src/components/PokemonItem/PokemonItem.tsx

```diff
@@ -0,0 +1,11 @@
+import { Pokemon } from "@/interfaces/pokemon";
+import "./pokemonItem.css";
+
+export const PokemonItem = ({ pokedexNumber, name }: Pokemon) => {
```

Comment:
```suggestion
export const PokemonItem = ({ pokemon }: PokemonItemProps) => {
```

WDYT?

That way we're consistent with naming our Props with actual Props suffix.

Reply:
Ok, the reason that I called it Pokemon and not PokemonProps is that I used this interface in more places in my code, and it is not a Prop in some places it is used. 

For example, in the PokemonList component there is a state that uses a Pokemon[] as a type for the state.
So not in every context this is a Prop.

but here it is a Prop...
So, I think a good solution here may be to import it specifically here "as" PokemonProp 

like this:

```bash
   import { Pokemon as PokemonProps } from "@/interfaces/pokemonProp";
```

this way here I can refer to it as  PokemonProps and in the rest of the app as simple Pokemon.

I`ll update this in my next commit:)

Reply:
When you have to update the Pokemon model, you might have some unexpected changes in this component.
This shouldn't be the case, and it should be decoupled even more.

Do you need to pass the whole Pokemon object here? Why not accept just what you need?

Reply:
This is a valid point I did not take into consideration (if the Pokemon model will be updateded). 
I will update this in my next commit to have its own interface and only pass the relevant fields from the object.